kubernetes/omsagent.yaml

```diff
@@ -443,6 +443,59 @@ spec:
                 - /opt/livenessprobe.sh
             initialDelaySeconds: 60
             periodSeconds: 60
+#Only in sidecar scraping mode
+        - name: omsagent-prometheus
+          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11092020"
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 500m
```

Comment:
1. isnt that having lower limits than other main container wouldnt  have impact main continer? 
2. what's the rationale behind picking these numbers? are you planning to share scale numbers?

Reply:
> 1. isnt that having lower limits than other main container wouldnt  have impact main continer?
> 2. what's the rationale behind picking these numbers? are you planning to share scale numbers?
>    Rashmi - #1. I dont see how it would affect the main container. Differernt containers in a pod can have diff limits.
>    #2. This was optimized for the 95th percentile for pods exposing ~ 40k to 60k metrics.

> # 1. Yes, different containers can have different limits. But, POD will be OOM killed for the lowest container. what I am saying, side car container which has lower limits can impact overall the POD. I think, we should have same limits for both containers.
# 2. 40 k to 60k metrics per minute? If you can share perf report for entire report and that will be helpful to understand and handle the incidents related to this.